app/models/challenge.rb

```diff
@@ -0,0 +1,42 @@
+class Challenge < ActiveRecord::Base
+  belongs_to :challenger, class_name: "Photo"
+  belongs_to :challenged, class_name: "Photo"
+
+  before_create :default_length
+  before_create :complete_false
+  after_update :results_email
+
+  def self.send_results
+    Challenge.where("ends_at < ? AND completed = ?", Time.now, false).each do |challenge|
+      challenge.complete!
+    end
+  end
+
+  private
+
+  def default_length
+    self.length ||= 6
+  end
+
+  def complete_false
+    self.complete = false
+    true
+  end
+
+  def results_email
+    if self.complete?
+      photo_challenger = self.challenger
+      photo_challenged = self.challenged
+      if self.votes_challenger == self.votes_challenged
+        ChallengeMailer.results_email(photo_challenger, 'tie')
```

Comment:
@levicole, take a look at [latest commit](https://github.com/iamkevinlowe/photochamp/commit/7207ba228e2521a2d23503196deebc433a1e07af). I think I did it right.